generators/server/templates/build.gradle.ejs

```diff
@@ -280,7 +280,7 @@ dependencies {
 
     // Use ", version: jhipsterDependenciesVersion, changing: true" if you want
     // to use a SNAPSHOT release instead of a stable release
-    implementation group: "tech.jhipster", name: "jhipster-framework"<% if (reactive) { %>, {
+    implementation group: "tech.jhipster", name: "jhipster-framework", version: jhipsterDependenciesVersion, changing: true<% if (reactive) { %>, {
```

Comment:
This needs to be reverted before merging.

Reply:
```suggestion
    implementation group: "tech.jhipster", name: "jhipster-framework"<% if (reactive) { %>, {
```